styles/styles.css

```diff
@@ -366,6 +366,14 @@ main img {
   padding: 0 1.2rem;
 }
 
+.blog-authors.columns > div > div.columns-img-col {
+  padding-bottom: 20px;
+}
+
+.blog-authors.columns > div {
+  font-size: 1.4rem;
+}
```

Comment:
Is there any reason for this CSS to be in the global `styles/styles.css`? looks like you are only styling only the columns block, so then you can put it in `blocks/columns/columns.css` with the rest of the CSS from this PR.

The rendering logic will make sure to load the columns CSS only if there is a columns block present on the page, thus increasing the performance. The global `styles/styles.css` will always be loaded.